### PR TITLE
A few more cosmetic improvements

### PR DESCRIPTION
**Pull request by alexreg**
Sorry, just a few more I forgot to include! Not big at all.

r? @gnzlbg

### CODE REVIEW

**Comment by gnzlbg** [comment]
Is there a way to automatically lint on these on CI ?

File: crates/core_arch/src/x86_64/bmi.rs [hunk 1]
```diff
@@ -170,7 +170,7 @@ mod tests {
 
     #[simd_test(enable = "bmi1")]
     unsafe fn test_blsr_u64() {
-        // TODO: test the behavior when the input is 0
+        // TODO: test the behavior when the input is 0.
```
Comment (gnzlbg) [inline crates/core_arch/src/x86_64/bmi.rs]: might want to write "`0`" here

**Comment by alexreg** [comment]
Done. Ta.

**Comment by alexreg** [comment]
> Is there a way to automatically lint on these on CI ?

Not that I know of, but would be nice.

**Comment by gnzlbg** [comment]
> Not that I know of, but would be nice.

Until we have one, what you are doing is an impossible battle to win: you can't fix all comments of all Rust contributors. 

I think it would be better to add a clippy lint that uses the system's spell and grammar checker (e.g. `aspell`) to correct spelling, grammar, etc. on comments, maybe with a way to control the spell-checker settings (locale, etc.). 

That would give different results on different systems, but we could choose to run clippy only on, e.g., Linux. Although if the spelling and grammar are correct on Linux, the chances that a Windows or MacOSX spell checker flag these as incorrect (and vice-versa) are probably slim as long as the locale's are configured to be the exact same.

**Comment by alexreg** [comment]
@gnzlbg You're not wrong... hopefully these changes have a least set a precedent though, so others will try to copy and adhere to the style.

**Comment by alexreg** [comment]
@gnzlbg You're not wrong... hopefully these changes have a least set a precedent though, so others will try to copy and adhere to the general style.

**Pull request merged**